src/components/map/mapControls/MapLayersControl.tsx

```diff
@@ -80,32 +84,33 @@ class MapLayersControl extends React.Component
                     />
                     <NetworkDateControl selectDate={this.selectDate}/>
                     <div className={s.sectionDivider} />
+                    <div className={s.inputTitle}>GEOMETRIAT</div>
                     <div className={s.checkboxContainer}>
                         <Checkbox
                             onClick={this.toggleMapLayerVisibility(MapLayer.link)}
                             checked={NetworkStore.isMapLayerVisible(MapLayer.link)}
-                            text={'Näytä alueen linkit'}
+                            text={'Alueen linkit'}
                         />
                     </div>
                     <div className={s.checkboxContainer}>
                         <Checkbox
                             onClick={this.toggleMapLayerVisibility(MapLayer.linkPoint)}
                             checked={NetworkStore.isMapLayerVisible(MapLayer.linkPoint)}
-                            text={'Näytä linkkien pisteet'}
+                            text={'Linkkien pisteet'}
                         />
                     </div>
                     <div className={s.checkboxContainer}>
                         <Checkbox
                             onClick={this.toggleMapLayerVisibility(MapLayer.node)}
                             checked={NetworkStore.isMapLayerVisible(MapLayer.node)}
-                            text={'Näytä alueen solmut'}
+                            text={'Alueen solmut'}
                         />
                     </div>
                     <div className={s.checkboxContainer}>
                         <Checkbox
-                            onClick={this.toggleMapLayerVisibility(MapLayer.nodeWithoutLink)}
+                            onClick={this.toggleMapLayerVisibility(MapLayer.nodeWithoutLink)} // tslint:disable
```

Comment:
Why is there an tslint here, and this tslint will disable tslinting for the whole file